docs/source/markdown/podman-volume-import.1.md

```diff
@@ -9,6 +9,7 @@ podman\-volume\-import - Import tarball contents into an existing podman volume
 ## DESCRIPTION
 
 **podman volume import** imports the contents of a tarball into the podman volume's mount point.
+The contents of the volume will be merged with the content of the tarball, the latter takes precedence.
```

Comment:
"with the latter taking precedence"

Reply:
Your version sounds better, I adopted it, thanks.